.github/CODEOWNERS

```diff
@@ -1,55 +1,203 @@
 # GitHub CODEOWNERS definition
 # See: https://help.github.com/articles/about-codeowners/
 
-# * @elastic/beats
+# The beats repository is owned by the @elastic/elastic-agent-data-plane Many teams contribute to it. The goal
+# is to cover all directories in the CODEOWNERS file. The list is sorted alphabetically by directory and sub directories.
```

Comment:
> The list is sorted alphabetically by directory and sub directories.

That's not true given the x-pack dirs that are listed near the beginning. Maybe we should drop most of the comments and do a `sort` on the contents.

Comments like `# All changes to the different deployments must be reviewed by data plane team.` do not add much value IMO, you can see that fact codified in `/deploy/ @elastic/elastic-agent-data-plane`.

Reply:
I unfortunately agree. When I started there were like 20 lines and it was easy to see the groups by team but then things exploded.

What I will do is merge the current PR as is but I want to follow up with automation which automatically will mean, sorting is in place and all comments will disappear.